docs/INSTALL.md

```diff
@@ -120,6 +121,10 @@ To start the application, either search for "Axolotl" in your app drawer or star
 
 `flatpak run org.nanuc.Axolotl`
 
+On systems without electron and Ubuntu Touch libraries:
+
+`flatpak run org.nanuc.Axolotl -e qml`
```

Comment:
In the .desktop file which is installed and used to start the application, we are already setting parameters.

Instead of setting this config at runtime, it can also be configured at install/buildtime.

Assuming this will not change, it might be an easier option to adjust it in the .desktop file before installing.

https://github.com/nanu-c/axolotl/blob/main/flatpak/org.nanuc.Axolotl.desktop#L5